cypress/integration/userCanNavigateThroughApplication.feature.js

```diff
@@ -0,0 +1,8 @@
+describe("User can navigate the app", () => {
+  beforeEach(() => {
+    cy.visit("http://localhost:3000");
```

Comment:
You don't need the full url address here if we set the "baseUrl" in the cypress.json file